.github/workflows/build.yml

```diff
@@ -34,8 +34,6 @@ jobs:
             coverage: true
     steps:
       - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
```

Comment:
I believe this was a remnant from when the version was based on tags, and so the full history had to be cloned